src/traffic_top/stats.h

```diff
@@ -62,6 +60,9 @@ const char end[]       = "\",\n";
 //----------------------------------------------------------------------------
 class Stats
 {
+  using string                            = std::string;
```

Comment:
Couldn't you have done using namespace std; in the class?  Looks uglier the just putting std:: in front of everything.

Reply:
C++ standard forbids it.
> A using-directive shall not appear in class scope

Another approach is adding `using namespace std;` at the beginning of each functions.

Reply:
Checked with below
```
 class Stats
 {
+  using namespace std;
```
```
traffic_top/stats.h:63:3: error: 'using namespace' is not allowed in classes
  using namespace std;
```